bucket/azcopy.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10",

```

Comment:
Put `version` in the first item, just like every other manifest (in this bucket) does.
```
    "version": "10.11.0",
    "description": "xxxxxxx",
    "homepage": "https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10",
    "license": "MIT",
```